src/main/webapp/pipe.js

```diff
@@ -311,124 +283,150 @@ function pipelineUtils() {
     }
 }
 
+function addPipelineHeader(html, component, data, c, resURL) {
+    html.push('<h1>' + htmlEncode(component.name));
+    if (data.allowPipelineStart) {
+        if (component.workflowComponent) {
+            html.push('&nbsp;<a id="startpipeline-' + c  +'" class="task-icon-link" href="#" onclick="triggerBuild(\'' + component.workflowUrl + '\', \'' + data.name + '\')">');
+        } else if (component.firstJobParameterized) {
+            html.push('&nbsp;<a id="startpipeline-' + c  +'" class="task-icon-link" href="#" onclick="triggerParameterizedBuild(\'' + component.firstJobUrl + '\', \'' + data.name + '\')">');
+        } else {
+            html.push('&nbsp;<a id="startpipeline-' + c  +'" class="task-icon-link" href="#" onclick="triggerBuild(\'' + component.firstJobUrl + '\', \'' + data.name + '\')">');
+        }
+        html.push('<img class="icon-clock icon-md" title="Build now" src="' + resURL + '/images/24x24/clock.png">');
+        html.push('</a>');
+    }
+    html.push('</h1>');
+}
+
+function checkDataForError(data, errrorDivId) {
+    var cErrorDiv = Q('#' + errrorDivId);
+    if (data.error) {
+        cErrorDiv.html('Error: ' + data.error).show();
+    } else {
+        cErrorDiv.hide().html('');
+    }
+}
+
+function isTaskLoggedToConsole(task) {
```

Comment:
Sounds much better, but I'd like to use "task" in status, because we check not only status, but also data. I have used your proposition from this comment: https://github.com/Diabol/delivery-pipeline-plugin/pull/262#discussion_r131482339